### PR TITLE
Add app source code for forms guide

### PR DESCRIPTION
**Pull request by sfc-gh-dmatthews**
## 📚 Context
Add source code files for apps embedded in /library/advanced-features/forms.

## 🧠 Description of Changes
Added python source code files for embedded apps.

## 💥 Impact
- [X] Not small

## 🌐 References
Notion: [Sprint task](https://www.notion.so/snowflake-corp/Doc-check-Press-enter-to-submit-widget-values-8f22d59abad344e2b1aa217e5c139c7d?pvs=4)

**Contribution License Agreement**
By submitting this pull request you agree that all contributions to this project are made under the Apache 2.0 license.

### CODE REVIEW

**Review by snehankekre — APPROVED** [review]
Approved pending caching changes

File: python/api-examples-source/forms.form_overview.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import streamlit as st
+import pandas as pd
+import numpy as np
+
+def get_data():
```
[the remark this answers is not in the document]
Comment (snehankekre) [reply python/api-examples-source/forms.form_overview.py:5]: Perhaps you've forgotten to push the changes? There's no additional commit with caching

File: python/api-examples-source/forms.form_overview.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    opacityB = row2[1].slider('B opacity', 20, 100, 50, label_visibility='hidden')
+    sizeB = row2[2].slider('B size', 50, 200, 100, step=10, label_visibility='hidden')
+
+    st.form_submit_button('Update map')
+
+alphaA = int(opacityA*255/100)
+alphaB = int(opacityB*255/100)
+
+df['color'] = np.where(df.team=='A',colorA+f'{alphaA:02x}',colorB+f'{alphaB:02x}')
+df['size'] = np.where(df.team=='A',sizeA, sizeB)
```
[the remark this answers is not in the document]
Comment (sfc-gh-dmatthews) [reply python/api-examples-source/forms.form_overview.py:41]: Updated:
```python
@st.cache_data
def color_map(df, colorA, alphaA, sizeA, colorB, alphaB, sizeB):
    df['color'] = np.where(df.team=='A',colorA+f'{alphaA:02x}',colorB+f'{alphaB:02x}')
    df['size'] = np.where(df.team=='A',sizeA, sizeB)
    return df

df = color_map(df, colorA, alphaA, sizeA, colorB, alphaB, sizeB)
```

File: python/api-examples-source/forms.form_overview.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (sfc-gh-dmatthews) [reply python/api-examples-source/forms.form_overview.py:5]: I was commenting as I was saving locally. Just pushed now. Sorry for the confusion.

**Pull request merged**